war/src/main/js/add-item.js

```diff
@@ -82,6 +78,65 @@ $.when(getItems()).done(function(data) {
       $('.input-help', context).removeClass('input-message-disabled');
     }
 
+    // About Scroll-linked effect: https://developer.mozilla.org/en-US/docs/Mozilla/Performance/Scroll-linked_effects
+    function doSticky() {
+      var decorator = $('form .footer .btn-decorator');
```

Comment:
:ant: It's a preference (and a commonly used convention) to use $ to start names for any variables containing the result of a jQuery selector